### PR TITLE
feat: replayq:is_mem_only/1

### PR DESCRIPTION
**Pull request by qzhuyan**

### CODE REVIEW

File: test/replayq_tests.erl [hunk 1]
```diff
@@ -301,6 +301,10 @@ comitter_crash_test() ->
       ok
   end.
 
+is_in_mem_test() ->
+  Q = replayq:open(#{mem_only => true}),
+  true = replayq:is_mem_only(Q).
```
Comment (thalesmg) [inline test/replayq_tests.erl]: Nit: maybe test the negative case?

**Review by thalesmg — APPROVED** [review]

**Comment by zmstone** [comment]
Need appup

File: test/replayq_tests.erl [hunk 2]
```diff
@@ -301,6 +301,20 @@ comitter_crash_test() ->
       ok
   end.
 
+is_in_mem_test_() ->
```
Comment (thalesmg) [inline test/replayq_tests.erl:304]:
```suggestion
is_in_mem_test() ->
```
?

**Review by thalesmg — APPROVED** [review]

File: test/replayq_tests.erl (hunk 2 above)
Comment (zmstone) [reply test/replayq_tests.erl:304]: `test_` is a test generator. https://www.erlang.org/doc/apps/eunit/chapter.html#writing-test-generating-functions

**Review by zmstone — APPROVED** [review]

File: test/replayq_tests.erl (hunk 2 above)
Comment (thalesmg) [reply test/replayq_tests.erl:304]: I see, didn't know about that!  Thanks for the reference!  :beers:

**Pull request merged**